packages/cli/src/commands/init.ts

```diff
@@ -78,11 +79,11 @@ export default class Init extends Command {
       description: 'Give the starter project name',
     }),
   };
-  private projectPath: string; //path on GitHub
-  private project: ProjectSpecBase;
-  private location: string;
-  private networkFamily: NETWORK_FAMILY;
-  private network: string;
+  private projectPath!: string; //path on GitHub
+  private project!: ProjectSpecBase;
+  private location!: string;
+  private networkFamily!: NETWORK_FAMILY;
+  private network!: string;
```

Comment:
I don't think we need some of these private variables. 

`networkFamily` and `network` are only used within `run`
`location` can be passed as an argument to `cloneCustomRepo`
I think similar changes could be made for `project` and `projectPath` too